guides/deployments/postgres.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Connect an External PostgreSQL"
+description: "Learn how connect Coder to an external postgreSQL database."
+---
+
+This guide walks through the steps to deploying Coder with an external
+PostgreSQL database connected.
+
+## Connecting to an external PostgreSQL
+
+By default, Coder will deploy a [TimescaleDB](https://www.timescale.com) inside
+the Kubernetes cluster used to install Coder. However, this is _only_
+recommended for evaluation purposes, as the database is not backed up, and can
+be lost if the cluster goes down.
+
+For those reasons, we recommend bringing in your own PostgreSQL for production
+deployments, and hosting it external to the cluster. Below are the steps to do
+so:
+
+1. Spin up a PostgreSQL instance if you have not already done so
```

Comment:
This step should be the bulk of the guide. This page does not otherwise provide much beyond the existing documentation for referencing a postgres instance in the helm.